BoxContentSDK/BoxContentSDK/Models/BOXFile.m

```diff
@@ -37,6 +37,22 @@ - (instancetype)initWithJSON:(NSDictionary *)JSONResponse
                                                    inDictionary:JSONResponse
                                                 hasExpectedType:[NSString class]
                                                     nullAllowed:NO];
+        
+        NSDictionary *fileVersionDictionary = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyFileVersion
+                                                                            inDictionary:JSONResponse
+                                                                         hasExpectedType:[NSDictionary class]
+                                                                             nullAllowed:YES
+                                                                       suppressNullAsNil:YES];
+        // Parse File Version.
+        if (fileVersionDictionary)
+        {
+            self.fileVersion = [[BOXFileVersionMini alloc] initWithJSON:fileVersionDictionary];
+        }
+        
+        self.fileVersion = [NSJSONSerialization box_ensureObjectForKey:BOXAPIObjectKeyVersionNumber
```

Comment:
The above seems correct. This line doesn't.

Reply:
Thanks. I fixed it.